references/gcp-sa-auth-shared-flow/README.md

```diff
@@ -0,0 +1,60 @@
+# GCP Service Account Authentication - Shared Flow
+
+This shared flow can be used obtain access tokens for Google Cloud service
+accounts. Access tokens are cached in the default environment cache resource for
```

Comment:
using the default environment cache means it cannot be manually cleared. what happens if tokens are revoked early? would we be stuck for 10 mins? it might be worth switching to a named cache.